### ISSUE TITLE
Could not find patch for MHIG_EU_SK_K1552

### ISSUE THREAD

**Issue by SimpleDOMno** [Mr-MIBonk/M.I.B._More-Incredible-Bash#440]
Unit went on update and no patch was found.
Maps updated db uodated but no fse codes availible.
08100009 tands as temporery.

Navigation unit is complaining about Nav databaae is not instaled or corrupt or missing.

Gem version 3.3 did not uodated after m.i.b install

**Comment by elbasan** [comment]
did you manage to fix this?

**Comment by SimpleDOMno** [comment]
no. probably some software glich. I asume original mapupdate file should work.
